e2e/pages/app/checkbox/checkbox.testpage.component.ts

```diff
@@ -1,8 +1,9 @@
-import { Component } from '@angular/core';
+import { Component, ChangeDetectionStrategy } from '@angular/core';
```

Comment:
Only a small thing but I've noticed that the dependencies are not organised alphabetically in lots of these files. In VSCode hit  [shift] + [alt] + [o] and it will do it automatically.